Contest_API.md

```diff
@@ -961,12 +961,12 @@ Properties of language objects:
 
 Properties of Command objects:
 
-| Name            | Type     | Description
-| :-------------- | :------- | :----------
-| command         | string   | Command to run.
-| args            | string ? | Argument list for command. `{files}` denotes where to include the file list. Defaults to empty string.
-| version         | string ? | Expected output from running the version-command. Defaults to empty string.
-| version-command | string ? | Command to run to get the version. Defaults to `<command> --version`.
+| Name             | Type     | Description
+| :--------------- | :------- | :----------
+| command          | string   | Command to run.
+| args             | string ? | Argument list for command. `{files}` denotes where to include the file list. Defaults to empty string.
+| version          | string ? | Expected output from running the version\_command. Defaults to empty string.
```

Comment:
```suggestion
| version          | string ? | Expected output from running the version command. Defaults to empty string.
```
Alternatively it could be quoted as `version\_command`